app/src/main/java/ip/team13/petowner/ui/activities/ActivitiesViewModel.kt

```diff
@@ -49,4 +54,9 @@ class ActivitiesViewModel(
 
             return items
         }
+
+    fun addActivity(activity: ActivityEntryModel) {
+        activityRepository.addActivity(activity)
+        notifyDataSetChange.invoke(activityData)
```

Comment:
The `invoke` call is not necessary. You can write just `lamba(parameters)` and it works the same